src/analytics.js

```diff
@@ -2,14 +2,16 @@ const path = require("path")
 const config = require('./config')
 const GoogleAnalyticsClient = require("./google-analytics/client")
 const GoogleAnalyticsDataProcessor = require("./process-results/ga-data-processor")
+const GoogleAnalyticsQueryBuilder = require("./google-analytics/query-builder")
 
 const query = (report) => {
   if (!report) {
     return Promise.reject(new Error("Analytics.query missing required argument `report`"))
   }
 
   return GoogleAnalyticsClient.fetchData(report).then(data => {
-    return GoogleAnalyticsDataProcessor.processData(report, data)
+  const query = GoogleAnalyticsQueryBuilder.buildQuery(report) // included here again because it doesn't get returned with data any longer
```

Comment:
The new client doesn't include the same reaponse structure as we had in UA so we can bring our built query in to include with report formating.